http-client-tls/Network/HTTP/Client/TLS.hs

```diff
@@ -122,7 +134,9 @@ convertConnection conn = makeConnection
 
 -- | Evil global manager, to make life easier for the common use case
 globalManager :: IORef Manager
-globalManager = unsafePerformIO (newManager tlsManagerSettings >>= newIORef)
+globalManager = unsafePerformIO $ do
+    man <- newManager $ mkManagerSettingsContext (Just globalContext) def Nothing
```

Comment:
IIUC this hunk is just expanding `>>=` to do-notation (fine) and inlining/copy-pasting `tlsManagerSettings`—why the inlining? If there's a tricky good reason, a comment would be useful, lest somebody tries removing the duplication.